Dockerfile

```diff
@@ -1,10 +1,13 @@
 FROM alpine:edge
-RUN ["apk", "add", "--no-cache", "python3", "python3-dev", "musl-dev", "linux-headers", "g++", "gmp-dev", "mpfr-dev", "mpc1-dev", "ca-certificates", "openblas-dev", "gfortran"]
-
-RUN ["mkdir", "/PySyft"]
-COPY requirements.txt /PySyft
+RUN apk add --no-cache \
+  python3 python3-dev \
+  g++ linux-headers \
+  musl-dev gmp-dev mpfr-dev mpc1-dev \
+  ca-certificates openblas-dev \
+  make gfortran
 
 WORKDIR /PySyft
-RUN ["pip3", "install", "-r", "requirements.txt"]
-COPY . /PySyft
-RUN ["python3", "setup.py", "install"]
+COPY . .
```

Comment:
what does this do?

Reply:
That copies the contents of the current folder (basically the git repo itself) from the docker build context to the image.